src/main/scala/gitbucket/core/service/IssuesService.scala

```diff
@@ -231,7 +231,7 @@ trait IssuesService {
    */
   def searchIssue(
     condition: IssueSearchCondition,
-    pullRequest: Boolean,
+    pullRequest: Option[Boolean],
```

Comment:
Hm, we need to express issues, pull requests or both in this parameter.

In that case, it might be better to introduce new sealed trait which is similar to enum in Java.

```scala
sealed trait IssueSearchOption

object IssueSearchOption {
  case object Issues extends IssueSearchOption
  case object PullRequests extends IssueSearchOption
  case object Both extends IssueSearchOption
}
```

Then, `searchIssue()` can take `IssueSearchOption` instead of `Option[Boolean]` as `searchOption`, for instance.

Reply:
@takezoe I introduced `IssueSearchOption ` at https://github.com/gitbucket/gitbucket/pull/2586/commits/5d3471c2934b4ade17f6b3f355013d6e177d02b2. but I'm not sure if it's ok like this...